Greenstand-Bilaws.md

```diff
@@ -0,0 +1,151 @@
+
+11.	REMOVAL OF DIRECTORS.
+
+A director may be removed by the vote of two-thirds of the remaining members. Absence from two consecutive, or any three meetings within a two-year term, shall be grounds for such removal.
+
+12.	RESIGNATION.
+
+A director may resign at any time by giving written notice to the board of directors, president, or secretary of the corporation. Unless otherwise specified in the notice, the resignation shall take effect upon receipt thereof by the board of directors or such officer, and the acceptance of the resignation shall not be necessary to make it effective.
+
+13.	COMPENSATION.
+
+No compensation shall be paid to directors, as such, for their services, but by resolution of the board of directors, a fixed sum and expenses for actual attendance at each regular or special meeting of the board of directors may be authorized, to include child care expense, mileage, and travel reimbursement. Expense reports are to be filed at a regular meeting and approved by the board. Nothing herein contained shall be construed to preclude any director from serving the corporation in any other capacity and receiving compensation therefor.
+
+14.	POWERS.
+
+Without limiting the general powers conferred by these bylaws and provided by law, the board of directors shall have, in addition to such powers, the following powers, namely:
+
+(a)	From time to time to make and change rules and regulations not inconsistent with law, or with these bylaws, for the management and control of the business of the corporation and its affairs, and of its officers, agents and employees; to lease, purchase or otherwise to acquire, in any lawful manner, for and in the name of the corporation, any and all real estate, personal property, letters patent, concessions, licenses, inventions, and other property rights or privileges whatsoever deemed necessary or convenient for the prose­cution of its business and which the corporation is authorized to acquire, and generally upon such terms and conditions as they think fit, and in their discretion to pay therefor, either wholly or partially, in any stocks, bonds, debentures, or other securities of the corporation.
+
+(b)	To sell or otherwise to dispose of any real estate, personal property, patents, licenses, inventions, property rights or privileges belonging to the corporation, whenever in their opinion its interest would be thereby promoted.
+
+(c)	To enter into agreements and contracts with individuals, groups of individuals, corporations or governments for any lawful purpose.
+
+(d)	To supervise and direct the officers, agents, and employees of the corporation and to see that their duties are properly performed.
```

Comment:
I guess most/all of the text is boilerplate for bylaws in Alaska. Looks generally appropriate, except for this line. GS being a volunteer organization, perhaps some special text should be added about managing and directing volunteers?

Reply:
Thanks, I've made a note for future edits.. 
These have been approved by the board and are now our official bylaws